packages/integration-tests/src/tests/console/rbac.test.ts

```diff
@@ -71,7 +71,7 @@ describe('RBAC', () => {
       text: `The API resource ${apiResourceName} has been successfully created`,
     });
 
-    await expect(page).toMatchElement('div[class$=header] div[class$=info] div[class$=name]', {
+    await expect(page).toMatchElement('div[class$=header] div[class$=info] div', {
```

Comment:
It looks like these changes have conflict with one of my recent PRs. But nevermind you can go first

Reply:
cool ty